Update Code using  ES6 features and Region tag Comments

priyankarp12 opened the pull request.

priyankarp12 committed:
Update quickstart.gs

priyankarp12 committed:
Update sendEmail.gs

priyankarp12 committed:
Update Code.gs 

Update region tag
Update comment

WorkspaceSamples reviewed (changes requested):
Please apps_script_gmail_send_emails to gmail_send_emails

WorkspaceSamples reviewed (changes requested):
Please add try catch

priyankarp12 committed:
Add try-catch block

priyankarp12 committed:
Added try-catch

priyankarp12 committed:
Added try-catch

priyankarp12 committed:
Added try-catch and update region comments 

1.Updated code using let and const
2.Added try-catch
3.Updated region tag comments
